user_modules/shared_testcases/testCasesForPolicyTable.lua

```diff
@@ -824,5 +834,230 @@ function testCasesForPolicyTable:Restore_preloaded_pt()
+
+		--[[Start get data from PTS]]
+		--TODO(istoimenova): function for reading INI file should be implemented
+        --local SystemFilesPath = commonSteps:get_data_form_SDL_ini("SystemFilesPath")
+        local SystemFilesPath = "/tmp/fs/mp/images/ivsu_cache/"
+        local file_pts = SystemFilesPath.."sdl_snapshot.json"
+
+		-- Check SDL snapshot is created correctly and get needed data 
+	    testCasesForPolicyTableSnapshot:create_PTS(true, {app_id}, {device_id}, {hmi_app_id})
+
+        --BasicCommunication.PolicyUpdate is sent immediatelly at triggering PTU. Should be checked in the scripts
+        -- EXPECT_HMICALL("BasicCommunication.PolicyUpdate", { file = file_pts, timeout = timeout_after_x_seconds, retry = seconds_between_retries})
+        -- :Do(function(_,data) self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {}) end)
+
+        local endpoints = {}
+        local is_app_esxist = false
+
+        for i = 1, #testCasesForPolicyTableSnapshot.pts_endpoints do
+          --using default URLs
+          if (testCasesForPolicyTableSnapshot.pts_endpoints[i].service == "0x07") then
+            endpoints[1] = { url = testCasesForPolicyTableSnapshot.pts_endpoints[i].value, appID = nil}
+          end
+        end
+        -- SDL.GetURLS => BasicCommunication.OnSystemRequest =>
+        -- SDL.OnStatusUpdate + 
+        -- app_id.SystemRequest => BasicCommunication.SystemRequest
+
+        local RequestId_GetUrls = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+        EXPECT_HMIRESPONSE(RequestId_GetUrls,{result = {code = 0, method = "SDL.GetURLS", urls = endpoints} } )
+        :Do(function(_,data) 
+
+      	  self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",{ requestType = "PROPRIETARY", fileName = ptu_file_name})
+	      EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY"})
+	      :Do(function(_,data)
+			
+		    EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATING"})
+		    :Do(function(_,data) 
+		    end)--EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATING"})
+
+			local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest",  {requestType = "PROPRIETARY", fileName = ptu_file_name}, ptu_file)
+			EXPECT_HMICALL("BasicCommunication.SystemRequest",{ requestType = "PROPRIETARY", fileName = SystemFilesPath..ptu_file_name })
+            :Do(function(_,data) 
+            	self.hmiConnection:SendResponse(systemRequestId,"BasicCommunication.SystemRequest", "SUCCESS", {})
+            	self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate", { policyfile = SystemFilesPath..ptu_file_name})
+            	EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UP_TO_DATE"})
+            end) --EXPECT_HMICALL("BasicCommunication.SystemRequest",{ requestType = "PROPRIETARY", fileName = ptu_file })
+            --TODO(istoimenova): SDL returns GENERIC_ERROR. Should be checked. As not in scope will be commented
```

Comment:
@Itileda Usually I remove them after final debug of tests, because is more easier to me at state of review process.
Now are removed because are used in common functions that should be reviewed asap.
Done in commit 73087db7aefd5db1652da6d8cb9c4b1c3d7469d9